src/PendingReason_Item.php

```diff
@@ -413,11 +414,29 @@ public static function handlePendingReasonUpdateFromNewTimelineItem(
             }
         }
 
+        if (!isset($new_timeline_item->input['pendingreasons_id']) && $new_timeline_item->getType() === 'TicketTask') {
```

Comment:
it doesn't matter what type of object is added (Followups / Task), if the pending reason is different it needs to be updated no?